### PR TITLE
make cats attack mouse toy

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
added cat faction to all cats so they attack mice and mouse toy
added mouse faction to mouse toy

## Why / Balance
piep

## Media
all cats attack the toy...
bingus doesnt attack since bingus isnt actually a cat

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- tweak: Cats will now play with mouse toys.

### CODE REVIEW

**Comment by TadJohnson00** [comment]
Does this fuck with medhuds? Do mouse toys actually appear alive?

**Comment by Bonktrauma** [comment]
Bingus is a sphynx, what are you on about

**Comment by deltanedas** [comment]
bingus is a simplemob

**Review by MilonPL — APPROVED** [review]
small change, everything looks good

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -666,6 +666,17 @@
     equippedPrefix: mouse
     slots:
     - HEAD
+  # Begin DeltaV additions: make cats attack mouse toys to play with them
+  - type: Damageable
+    damageContainer: Inorganic # not actually destructible, just lets it be attacked
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml]: oh fuck the diagnostic hud shows health bars/icons for inorganic and silicon containers, you'd want to use `StructuralInorganic` instead

**Review by MilonPL — CHANGES_REQUESTED** [review]
minor oversight

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Fun/toys.yml]: yes its fixed now

**Review by MilonPL — APPROVED** [review]

**Pull request merged**